package.json

```diff
@@ -120,6 +120,9 @@
 			"rule-empty-line-before": null,
 			"at-rule-empty-line-before": null,
 			"selector-max-compound-selectors": null
-		}
+		},
+		"ignoreFiles": [
+			"**/*.md"
+		]
```

Comment:
This doesn't seem to change the linter's output

Reply:
I don't know why but without this option [the VS Code `stylelint` extension](https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint) displays all sorts of errors in the `readme.md` file as if it was a stylesheet.